src/main/java/seedu/address/model/room/RoomList.java

```diff
@@ -76,6 +97,15 @@ public void addRooms(int numOfRooms) {
         this.numOfRooms = numOfRooms;
         addRooms();
     }
+
+    /**
+     * Adds this room to the RoomList
+     * @param room is added to RoomList
+     */
+    public void addRooms(Room room) {
+        rooms.add(room);
+        roomObservableList.add(room);
+    }
     /**
      * Sets the elements of {@code roomObservableList}.
```

Comment:
can delete the method below. i dont think it's in use anymore. i only created it to bypass the need for an observablelist

Reply:
I need it for the Json.